### ISSUE TITLE
use variables in error messages

### ISSUE THREAD

**Issue by wesley3523** [kelp404/angular-validator#46]
Is it possible to use some variable in the error message, something like

error: 'This field must have {number} characters'

i appreciate it.

**Comment by huei90** [comment]
yes, it can. please search the documentation and issue.

**Comment by wesley3523** [comment]
Yes i did, but can not find the awnser. maybe some example can help?

Thanks
